src/hotspot/share/opto/library_call.cpp

```diff
@@ -6789,11 +6789,20 @@ bool LibraryCallKit::inline_galoisCounterMode_AESCrypt() {
   Node* state_start = array_element_address(state, intcon(0), T_LONG);
   Node* subkeyHtbl_start = array_element_address(subkeyHtbl, intcon(0), T_LONG);
 
+  ciKlass* klass = ciTypeArrayKlass::make(T_LONG);
+  Node* klass_node = makecon(TypeKlassPtr::make(klass));
+
+  Node* avx512_subkeyHtbl = new_array(klass_node, intcon(96), 0);
+  if (avx512_subkeyHtbl == NULL) return false;
```

Comment:
This looks very Intel-specific, but it's in generic code. Please make this constant 96 a symbol and push it into a header file in the x86 back end.

Reply:
I'll modify the code. Thanks.